doc/nrf/known_issues.rst

```diff
@@ -2102,113 +2325,29 @@ FW upgrade is broken for multi-image builds
-
-.. rst-class:: v2-2-0
-
-NCSDK-19156: Building SPM for other boards than ``nrf5340dk_nrf5340_cpuapp`` and ``nrf9160dk_nrf9160`` fails with compilation error in cortex_m_systick.c
-  This happens because the :kconfig:option:`CONFIG_CORTEX_M_SYSTICK` configuration option is enabled while the systick node is disabled in the devicetree.
-
-  **Workaround** Enable the systick node in a DTS overlay file for the SPM build by completing the following steps:
-
-  1. Create an overlay file :file:`systick_enabled.overlay` with the following content:
-
-     .. code-block::
-
-        &systick {
-          status = "okay";
-        };
-
-  #. Add the overlay file as a build argument to SPM:
-
-     .. parsed-literal::
-       :class: highlight
-
-       west build -- -Dspm_DTC_OVERLAY_FILE=systick_enabled.overlay
-
-.. rst-class:: v2-2-0 v2-1-4 v2-1-3 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0 v1-8-0 v1-7-1 v1-7-0 v1-6-1 v1-6-0 v1-5-2 v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0 v1-3-2 v1-3-1 v1-3-0
-
-NCSIDB-114: Default logging causes crash
-  Enabling default logging in the Secure Partition Manager sample makes it crash if the sample logs any data after the application has booted (for example, during a SecureFault, or in a secure service).
-  At that point, RTC1 and UARTE0 are non-secure.
-
-  **Workaround:** Do not enable logging and add a breakpoint in the fault handling, or try a different logging backend.
+The issues in this section are related to :ref:`ecosystem_integration`.
 
-.. rst-class:: v1-6-1 v1-6-0 v1-5-2 v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0 v1-3-2 v1-3-1 v1-3-0 v1-2-1 v1-2-0 v1-1-0
+Pelion
+------
 
-NCSDK-8232: Secure Partition Manager and application building together
-  It is not possible to build and program Secure Partition Manager and the application individually.
+The issues in this section are related to Pelion Device Management integration.
+The support for Pelion Device Management and the related library and application were removed in :ref:`nRF Connect SDK v1.9.0 <ncs_release_notes_190>`.
 
-.. rst-class:: v1-5-2 v1-5-1 v1-5-0
+.. rst-class:: v1-6-1 v1-6-0
 
-CIA-248: Samples with default SPM config fails to build for ``thingy91_nrf9160_ns``
-   All samples using the default SPM config fails to build for the ``thingy91_nrf9160_ns``  build target if the sample is not set up with MCUboot.
+NCSDK-10196: DFU fails for some configurations with the quick session resume feature enabled
+  Enabling :kconfig:option:`CONFIG_PELION_QUICK_SESSION_RESUME` together with the OpenThread network backend leads to the quick session resume failure during the DFU packet exchange.
+  This is valid for the :ref:`nRF52840 DK <ug_nrf52>` and the :ref:`nRF5340 DK <ug_nrf5340>`.
 
-   **Workaround:** Use the main branch.
+  **Workaround:** Use the quick session resume feature only for configurations with the cellular network backend.
 
 MCUboot
```

Comment:
Should we move this to the end, next to Zephyr, TF-M etc.?

Reply:
Let's see how it's gonna look.